docs/headless-cms/docs/06-02-clusterdocstopic.md

```diff
@@ -0,0 +1,86 @@
+---
+title: ClusterDocsTopic
+type: Custom Resource
+---
+
+The `clusterdocstopics.cms.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define an orchestrator that creates ClusterAsset CRs for a specific asset type. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd clusterdocstopics.cms.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample ClusterDocsTopic custom resource (CR) that provides details of the ClusterAsset CRs for the **redis**, **asyncapi**, **markdown**, and **openapi** source types.
+
+```
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  name: service-catalog
+  labels:
+     viewContext: docs-view
+     groupName: components
+spec:
+  displayName: "Service Catalog"
+  description: "Service Catalog documentation"
+  sources:
+    redis:
+      mode: package
+      url: https://github.com/kyma-project/bundles/releases/download/latest/redis-0.0.3.tgz
+    asyncapi:
+      mode: single
+      url: https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/1.2.0/slack-rtm.yml
+    markdown:
+      mode: package
+      url: https://github.com/kyma-project/kyma/archive/master.zip
+      filter: ^kyma-master/docs/
+    openapi:
+      mode: single
+      url: https://petstore.swagger.io/v2/swagger.json
+status:
+  lastHeartbeatTime: "2019-03-18T13:42:55Z"
+  message: Assets are ready to use
+  phase: Ready
+  reason: AssetsReady
+
+```
+
+## Custom resource parameters
+
+This table lists all possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Mandatory      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** |    **YES**   | Specifies the name of the CR. |
+| **metadata.labels** |    **YES**   | Specifies how to filter and group ClusterAsset CRs that the ClusterDocsTopic CR defines. The available labels include **viewContext** and **groupName**. |
```

Comment:
explain that those specific labels like `cms.kyma-project.io/viewContext` is our convention that we applied to have easy filtering of content for the Console UI purposes, explain that user of the component can define his own convention that is suitable for his usecase

Reply:
I think it would also make sense to have a separate details document about it, so you can just link there instead of repeating the same info in 2 different docs about custom resources. It could be called `Documentation filtering` and it would explain that CR spec doesn't have any params that could be used for filtering as this responsibility is left on the `labels` and the user that depending on his use case can define a different labeling convention. Here we can provide explanation about how we implemented labels convention for our needs, for filtering documentation that is displayed in the console UI

Reply:
remember that convention doesn't applu to key like `cms.kyma-project.io/viewContext` but also it's value, so we have for now 2 values supported, `docs-ui` and `service-catalog`

Reply:
<tbc; comments 2 & 3>

Reply:
Actually, I created a separate doc for that.